go/libkb/services.go

```diff
@@ -15,6 +15,10 @@ import (
 //=============================================================================
 
 func MakeProofChecker(c ExternalServicesCollector, l RemoteProofChainLink) (ProofChecker, ProofError) {
+	if c == nil {
+		return nil, NewProofError(keybase1.ProofStatus_UNKNOWN_TYPE,
```

Comment:
When do you wind up in this case, on mobile?  You might be missing an initialization step...